flexget/plugins/list/sonarr_list.py

```diff
@@ -29,7 +29,11 @@ class SonarrSet(MutableSet):
             'include_data': {'type': 'boolean', 'default': False},
             'search_missing_episodes': {'type': 'boolean', 'default': True},
             'ignore_episodes_without_files': {'type': 'boolean', 'default': False},
-            'ignore_episodes_with_files': {'type': 'boolean', 'default': False}
+            'ignore_episodes_with_files': {'type': 'boolean', 'default': False},
+            'profile_id': {'type': 'number', 'default': 1},
+            'season_folder': {'type': 'boolean', 'default': False},
+            'monitored': {'type': 'boolean', 'default': True},
+            'rootFolderPath': {'type': 'number', 'default': 1}
```

Comment:
We usually use snake case and not camel case even if it's a "keyword".

Reply:
@cvium Never even heard of snake case before.. Guessing snake case is like "profile_id" for example?

Reply:
Yes

Reply:
This has now been taken care of.